pkg/routes/v1.go

```diff
@@ -22,7 +22,7 @@ func loadV1Routes(r *gin.Engine, h *handler.Handler, repo store.DBRepo, s *store
 	v1.PUT("/profile", amw.WithAuth, h.Profile.UpdateInfo)
 
 	// employees
-	v1.GET("/employees", amw.WithAuth, pmw.WithPerm("employees.read"), h.Employee.List)
+	v1.GET("/employees", amw.WithAuth, h.Employee.List) //pmw.WithPerm("employees.read"),
```

Comment:
why?

Reply:
just for developing purpose, I updated it